### DISCUSSION TITLE
Jira Integration

### DISCUSSION THREAD

**Discussion by DevilsHome** [Syslifters/sysreptor#157] [General]
I use jira for tracking is it possible to create jira integration to track findings with Assignment and Status change using jira api

**Comment by aronmolnar** [comment]
Hi there,
We had the same request yesterday from another user.

That's not (yet) available.
If you prepare us REST API calls for Jira (using dummy data; best if already in Python), we can do the integration on our side.

We would use our CLI tool for this (https://docs.sysreptor.com/cli/getting-started/)

**Comment by wrathfulDiety** [comment]
Hello!

I'm that other user :D Please keep me in the loop on how this goes :)

**Reply by aronmolnar** [reply]
All clear 😂

**Comment by unixerius** [comment]
Keeping an eye on this one, as we have customers using both TopDesk and Jira. It'd be great to push findings from Sysreptor into either of those. Luckily both have a decent API.

**Comment by pissi96** [comment]
We are also keeping an eye on the Jira integration, as it is an essential part of our work environment. We are eagerly awaiting further developments. Thank you.
